analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseDateTimeInsteadOfDateTimeOffset.CSharp9.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+DateTime myDate = new(1); // Noncompliant {{Prefer using "DateTimeOffset" struct instead of "DateTime"}}
+//                ^^^^^^
+_ = new DateTime(1, 1, 1, 1, 1, 1, 1, 1, new GregorianCalendar());                   // Noncompliant
+_ = new DateTime(1, 1, 1, 1, 1, 1, 1, 1, DateTimeKind.Utc);                          // Noncompliant
+_ = new DateTime(1, 1, 1, 1, 1, 1, 1, 1, new GregorianCalendar(), DateTimeKind.Utc); // Noncompliant
+
+_ = DateTime.UnixEpoch; // Noncompliant
+Span<char> span = new();
```

Comment:
What do these 3 lines test?

Reply:
Constructor not recognized in .NET Framework, [UnixEpoch](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.unixepoch) introduced in Core 2.1 or Standard 2.1 and the last span was initialized to test the [TryFormat](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.tryformat) method

Reply:
`UnixEpoch` is fine, but the 3 lines following that will not trigger the rule. If you want to test whether the rule will raise an issue on instance method calls, then that can be done in the common (.NET Framework) test cases.

Reply:
Wait I thought with "3 lines" you were referring to 8 10 and 11 (the one highlighted in the comments above).
If you are referring to 11 12 and 13(?) then 11 is just to create the span needed for the TryFormat method line 12 (we are only making sure that this is indeed compliant). Line 13 is just AddMicroseconds method, not supported in .NET Framework